Automattic/wp-calypso
# Signup reskin: Add the side section for the domain step

#### Changes proposed in this Pull Request

This PR is towards implementing the reskinned signup experiment. For more context on the experiment, read pbxNRc-IO-p2.

NOTE:

This PR implements only the sidebar section of the domain step. The other changes will be implemented in subsequent PRs.

Here's the [Figma file](https://www.figma.com/file/gnYBUOBthTMckJRJr3pZmk/Control-Experiments?node-id=1114%3A536) for reference.

For the record:
Parent PR: https://github.com/Automattic/wp-calypso/pull/51483 and root PR: https://github.com/Automattic/wp-calypso/pull/50512

<img width="1256" alt="Screenshot 2021-03-31 at 4 04 45 PM" src="https://user-images.githubusercontent.com/1269602/113141683-a09e6300-9247-11eb-9462-9c49536bae19.png">


#### Testing instructions



* Assign yourself to treatment from Abacus, and visit /start.
* Proceed to the domains step and verify that the "sidebar" info section in the domains step matches the [Figma design](https://www.figma.com/file/gnYBUOBthTMckJRJr3pZmk/Control-Experiments?node-id=1114%3A536).
* Clicking on the CTA "View plans" should take to the plans step screen. The "Start with free" button should be hidden in that page.
* Clicking on the CTA "Use a domain I own" should take to the mappings/transfer page
* Start over and verify that control group experience is unaffected
